qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -450,23 +441,51 @@ void allowsOnlyProperlyOrderedAtClauses() throws Exception {
      * @throws Exception If something wrong happens inside
      */
     @Test
-    @Disabled
-    void passesWindowsEndsOfLineWithoutException() throws Exception {
-        this.validate("WindowsEol.java", false, "LICENSE found:");
+    void prohibitWindowsEndsOfLine() throws Exception {
+        final String file = "WindowsEol.java";
+        final Collection<Violation> results = this.runValidation(file, false);
+        final String message = "Lines in file should end with Unix-like end of line";
+        final String name = "RegexpMultilineCheck";
+        MatcherAssert.assertThat(
+            results,
+            Matchers.contains(
+                new ViolationMatcher(
+                    "Expected line ending for file is LF(\\n), but CRLF(\\r\\n) is detected",
+                    file, "1", "NewlineAtEndOfFileCheck"
```

Comment:
It looks strange for me, that file has 10 lines of code and we are expecting NewlineAt**EndOfFile**Check at line 1